README.md

```diff
@@ -108,10 +108,27 @@ If `swiftc` is not in your `PATH`, you must specify the path to it using
 `-D CMAKE_Swift_COMPILER=`.
 
 ```
-cmake -G Ninja -B out -S swift-apis
+cmake -B out -D USE_BUNDLED_CTENSORFLOW=YES -D BUILD_X10=YES -G Ninja -S swift-apis
 cmake --build out
 ```
 
+If you are intending to develop X10, we recommend that you pass the
+`-no-stdlib-rpath` flag during compilation:
+
+```
+cmake -B out -D BUILD_X10=YES -D USE_BUNDLED_CTENSORFLOW=YES -D CMAKE_Swift_FLAGS="-no-stdlib-rpath" -G Ninja -S swift-apis
+cmake --build out
+```
+
+If you are not intending to develop X10, we recommend that you use the bundled
+X10 by passing `-D USE_BUNDLED_X10=YES`:
+
+```
+cmake -B out -D BUILD_X10=YES -D USE_BUNDLED_CTENSORFLOW=YES -D USE_BUNDLED_X10=YES -D CMAKE_Swift_FLAGS="-no-stdlib-rpath" -G Ninja -S swift-apis
```

Comment:
It looks like you're passing `-no-stdlib-rpath` here as well. Is that intentional? If so, it seems like we should set the default `CMAKE_Swift_FLAGS` to include `-no-stdlib-rpath`, right/

Reply:
Oops, no, that should not be there.  Copy-pasta.

Reply:
Why both `-D BUILD_X10=YES` and `-D USE_BUNDLED_X10=YES` together?

Reply:
The BUILD_X10 currently also switches between the tensorflow and x10 support in swiftTensorFlow.  The bundled version allows you to avoid building the x10 library from source.  Once we remove the option to build without x10, we should be able to consolidate the options.

Reply:
This combination throws a series of errors with the latest nightly.
```
$ cmake -B out -D BUILD_X10=YES -D USE_BUNDLED_CTENSORFLOW=YES -D USE_BUNDLED_X10=YES -D CMAKE_Swift_COMPILER=/home/michellecasbon/tmp/swift-tensorflow-DEVELOPMENT-ubuntu18.04/usr/bin/swiftc -G Ninja -S swift-apis
-- The CXX compiler identification is GNU 7.5.0
-- The Swift compiler identification is Apple 5.3
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Check for working Swift compiler: /home/michellecasbon/tmp/swift-tensorflow-DEVELOPMENT-ubuntu18.04/usr/bin/swiftc
-- Check for working Swift compiler: /home/michellecasbon/tmp/swift-tensorflow-DEVELOPMENT-ubuntu18.04/usr/bin/swiftc -- works
-- Could NOT find X10 (missing: X10_LIBRARY)
-- Configuring done
CMake Error at Sources/TensorFlow/CMakeLists.txt:77 (target_link_libraries):
  Error evaluating generator expression:

    $<TARGET_FILE_DIR:x10>

  No target "x10"
....
```